lib/Search/ElasticSearch/ElasticSearchEngine.php

```diff
@@ -88,7 +82,7 @@ public function search(SearchQuery $query): SearchResults
         $end = microtime(true);
         $searchTime = ($end - $start);
 
-        return new SearchResults($results, true, $searchTime, $hits['hits']['total']);
+        return new SearchResults($results, true, $searchTime, $hits['hits']['total']['value']);
```

Comment:
Hi @Dillon-Brown is this related with the index change? does seem so? Has the structure changed on elastic search?

Reply:
Yes structure has changed on ElasticSearch, I've also changed what comes back from ```parseHits``` as it now needs to include ```module_source``` in it's array so that we can determine the index.